course/layouts/shortcodes/resource_link.html

```diff
@@ -0,0 +1,7 @@
+{{- $uuid := index .Params 0 -}}
+{{- $title := index .Params 1 -}}
+{{- range $.Site.Pages -}}
+  {{- if eq (replace .Params.uid "-" "") (replace $uuid "-" "") -}}
+    <a href="{{- .Permalink -}}">{{ $title }}</a>
+  {{- end -}}
+{{- end -}}
```

Comment:
do you think there's anything to be gained by using the [where function](https://gohugo.io/functions/where/) here?

Reply:
Maybe but it doesn't look like there's any way to do the `replace .Params.uid "-" ""` part in a where function

Reply:
true...do we have wandering dashes that we need to worry about? it somewhat takes away from the 'uuid-ness' of a uuid if it can't just do a direct comparison haha

Reply:
I think the uid from ocw-to-hugo doesn't have dashes and the uuid from ocw-studio does... we should probably add dashes in ocw-to-hugo to make that more consistent

Reply:
Actually since we aren't exporting resource shortcodes in ocw-to-hugo yet we can probably just do a direct comparison here